### PR TITLE
Try to design map items

### PR DESCRIPTION
**Pull request by iiLubos**
MapItems
Map objects:
- clickable: MMMapButton {}
- text based: MMMapLabel { text: "20.0 m" }

### CODE REVIEW

**Comment by iiLubos** [comment]
Ready for review

**Review by PeterPetrik — COMMENTED** [review]
LGTM

File: app/qmlV2/component/MMMapLabel.qml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Controls.Basic
+import Qt5Compat.GraphicalEffects
```
Comment (tomasMizera) [inline app/qmlV2/component/MMMapLabel.qml:13]: Seems not to be needed

File: app/qmlV2/component/MMMapLabel.qml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import "../Style.js" as Style
+import "."
+
+Item {
+  id: control
+
+  width: text.width
+  height: Style.commonItemHeight
```
Comment (tomasMizera) [inline app/qmlV2/component/MMMapLabel.qml:21]: What is "commonItemHeight"? Let's be more specific and have `mapItemHeight`

File: app/qmlV2/component/MMMapLabel.qml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  visible: control.text.length > 0
```
Comment (tomasMizera) [inline app/qmlV2/component/MMMapLabel.qml:22]: Let's not bind the visible state, what is it good for? :)

File: app/qmlV2/component/MMMapLabel.qml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    layer.effect: MMShadow {}
+
+    Text {
+      id: text
+
+      anchors.centerIn: parent
+      color: control.textColor
+      text: control.text
+      font: Qt.font(Style.t3)
+      leftPadding: Style.commonSpacing
```
Comment (tomasMizera) [inline app/qmlV2/component/MMMapLabel.qml:46]: Let's not use "common" spacing. It will backfire later. Let's have specific spacing for map items

**Review by tomasMizera — CHANGES_REQUESTED** [review]
See comments :)

**Comment by iiLubos** [comment]
Applied all the review comments.
Support for an icon inside the Map label.

**Review by tomasMizera — CHANGES_REQUESTED** [review]
Bla bla test

**Review by tomasMizera — APPROVED** [review]

**Pull request merged**